domains/anonaddy.is-local.org.json

```diff
@@ -0,0 +1,16 @@
+{
+  "description": "Personal website with email forwarding on custom domain",
+
+  "domain": "is-local.org",
+  "subdomain": "anonaddy",
+
+  "owner": {
+    "email": "wolfo89+foexo@protonmail.com"
+  },
+
+  "record": {
+    "NS": ["ns1.zilore.net", "ns2.zilore.net"]
```

Comment:
We no longer support NS records as per readme.md:

![image](https://github.com/open-domains/register/assets/76603072/29102908-9d0e-4f3a-9483-a9361c3c2575)